File: SelectionSort/SelctionSort-Java/src/Sort/Sort.java
package Sort;

public class Sort 
{
  public void selection_sort(int[] arr)
  {
    int len=arr.length;

    for(int i=0;i<len;i++)
    {
      int minIndex=i;
      for(int j=i+1;j<len;j++)
      {
        if(arr[j]<arr[minIndex])
        {
          minIndex=j;
        }
      }
      int temp=arr[i];
      arr[i]=arr[minIndex];
      arr[minIndex]=temp;
    }
  }
}

File: SelectionSort/SelectionSort-Csharp/src/Sort.cs
using System;
using System.Collections.Generic;

namespace Sort 
{
  class Sort 
  {
    public void selection_sort(List<int>arr)
    {
      int n=arr.Count;
      
      for(int i=0;i<n;i++)
      {
        for(int j=i+1;j<n;j++)
        {
          int min=i;
 
          if(arr[j]<arr[min])
          {
            min=j;
          }

          int temp=arr[i];
          arr[i]=arr[min];
          arr[min]=temp;
        }
      }
    }
  }
}
